swatch-contracts/src/main/java/com/redhat/swatch/contract/resource/ContractsTestingResource.java

```diff
@@ -340,7 +339,17 @@ public OfferingResponse syncOffering(String sku) throws ProcessingException {
             throw new InternalServerErrorException(apiException.getMessage());
         }
       }
-      response.setDetail("Error syncing offering");
+      if (e instanceof NotFoundException
```

Comment:
this `if` statement is no longer necessary, right?

Reply:
It's in case we get another type of runtime exception of an unexpected type and converting to a 500. If you are good with just throwing it as is, then I can take it out.

Reply:
500 makes sense to me in this scenario. +1 to drop it